main.js

```diff
@@ -8,12 +8,24 @@ var settings = require("./src/javascript/core/settings");
  * @type {string}
  */
 var version = '0.0.24';
+/**
+ * The source of this event.
+ * @type {string}
+ */
+var source = 'o-tracking';
+/**
+ * The API key.
+ * @type {string}
+ */
+var api_key = 'qUb9maKfKbtpRsdp0p2J7uWxRPGJEP';
```

Comment:
Yes, almost intentional!

It will allow us to rotate keys in the future and stop getting requests from some websites who like to copy our content.